app/routes/index.js

```diff
@@ -1,17 +1,15 @@
 const config = require('../config')
-const session = require('../session')
-const { requestAuthorizationCodeUrl } = require('../auth')
 
 module.exports = {
   method: 'GET',
   path: `${config.urlPrefix}/start`,
   options: {
     auth: false,
     handler: async (request, h) => {
-      return h.view('index', {
-        defraIdLogin: requestAuthorizationCodeUrl(session, request),
-        ruralPaymentsAgency: config.ruralPaymentsAgency
-      })
+      // old world apply is no longer active, redirect user to new world start
+      // it's not actually possible to navigate directly to this page anyway anymore, except
+      // if the user decided to type it in url themselves, or had a bookmark saved here
+      return h.redirect(`${config.urlPrefix}/endemics/start`)
```

Comment:
I do tend to support [Defra guidance on comments](https://defra.github.io/software-development-standards/standards/common_coding_standards/). Wonder if the suggested principle of 'separate in order to name' could be applied here to provide similar context. Something like:

```
const deprecatedOldEntryPoint =  `${config.urlPrefix}/start`
const newAppEntryPoint = `${config.urlPrefix}/endemics/start`

module.exports = {
  url: deprecatedOldEntryPoint,
  handler: (_, h) => h.redirect(newAppEntryPoint)
}

```

Reply:
Willing to compromise - see new changes